### PR TITLE
Fix inconsistent grammar in device sign out modal

### PR DESCRIPTION
**Pull request by EECvision**
Fixes https://github.com/vector-im/element-web/issues/21699

Corrected a grammar inconsistency in the device sign out modal.

Signed-off-by: Emmanuel Ezeka eec.studies@gmail.com

---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

Preview: https://pr8253--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Review by gsouquet — DISMISSED** [review]

**Comment by gsouquet** [comment]
I'm not sure I follow the inconsistency you've raised here. 
If you could give a bit more context that'd be good, otherwise I will close the pull request

**Comment by EECvision** [comment]
> I'm not sure I follow the inconsistency you've raised here. If you could give a bit more context that'd be good, otherwise I will close the pull request

Alright. 

let me make it more detailed

File: src/i18n/strings/en_EN.json [hunk 1]
```diff
@@ -1226,7 +1226,7 @@
     "Confirm logging out these devices by using Single Sign On to prove your identity.|one": "Confirm logging out this device by using Single Sign On to prove your identity.",
     "Confirm signing out these devices": "Confirm signing out these devices",
     "Click the button below to confirm signing out these devices.|other": "Click the button below to confirm signing out these devices.",
-    "Click the button below to confirm signing out these devices.|one": "Click the button below to confirm signing out this device.",
+    "Click the button below to confirm signing out these devices.|one": "Click the button below to confirm signing out these devices.",
```
[the remark this answers is not in the document]
Comment (EECvision) [reply src/i18n/strings/en_EN.json:1229]: Okay. The issue pointed at could not have been valid. Or, was it referring to a different thing.

see https://github.com/vector-im/element-web/issues/21699

**Comment by EECvision** [comment]
> > I'm not sure I follow the inconsistency you've raised here. If you could give a bit more context that'd be good, otherwise I will close the pull request
> 
> Alright.
> 
> let me make it more detailed

A screenshot of what I changed.

error1

matrix-react-sdk\src\i18n\strings\en_EN.json Ln:1229 Col: 124

Maybe, I should change the line above Ln: 1229 instead to be singular. Will that be ok?

**Comment by luixxiul** [comment]
@EECvision This is not related to the issue you work on, but I would like to recommend you to have a look at https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue.

I'd write: `This PR fixes https://github.com/vector-im/element-web/issues/21699`.

File: src/i18n/strings/en_EN.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (EECvision) [reply src/i18n/strings/en_EN.json:1229]: yeah, thank you.

**Comment by EECvision** [comment]
> @EECvision This is not related to the issue you work on, but I would like to recommend you to have a look at https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue.
> 
> I'd write: `This PR fixes https://github.com/vector-im/element-web/issues/21699`.

Okay, thanks.

File: src/i18n/strings/en_EN.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (EECvision) [reply src/i18n/strings/en_EN.json:1229]: I have found the solution. I was anxious about solving the problem that I did not look well before making my first push.

**Modal for signing out a single device.**
error1

**Modal for signing out multiple devices.**
error2


I will make my push as soon as I finish translating the change in other languages.

[the remark this answers is not in the document]
Comment (EECvision) [reply src/i18n/strings/en_EN.json]: Hi @gsouquet, I have updated my changes. Thank you.

**Comment by SimonBrandner** [comment]
@EECvision, if you put `Fixes https://github.com/vector-im/element-web/issues/21699` into the PR description, the PR will automatically close the issue when merged, could you please do that?

**Comment by EECvision** [comment]
> @EECvision, if you put `Fixes https://github.com/vector-im/element-web/issues/21699` into the PR description, the PR will automatically close the issue when merged, could you please do that?

Thank you I have updated the description.

**Review by t3chguy — CHANGES_REQUESTED** [review]
Please do not change any i18n files by hand except en_EN, it'll create conflicts in Weblate otherwise. All translations must be done via Weblate.

**Review by t3chguy — APPROVED** [review]
Looks good to me, thanks for your contribution!

**Review by t3chguy — CHANGES_REQUESTED** [review]
Needs DCO sign-off https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.md#sign-off

**Comment by EECvision** [comment]
> Needs DCO sign-off https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.md#sign-off

I have added a sign off to the PR description. Thank you

**Review by t3chguy — APPROVED** [review]
Thanks!

**Pull request merged**